### PR TITLE
Add Prometheus rule for init node deployment failures

### LINKED ISSUE #3041: Create alert for init_node_deployment_failures metric
**User Story** Alert for `init_node_deployment_failures` should be created once #2989 gets merged.

**Acceptance criteria**
- [ ] There is an alert for `init_node_deployment_failures` metric.

### PR DESCRIPTION
**Pull request by maciaszczykm**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3041.

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: config/monitoring/prometheus/rules/kubermatic-master-kubermatic.yaml [hunk 1]
```diff
@@ -19,3 +19,10 @@ groups:
     for: 15m
     labels:
       severity: warning
+  - alert: KubermaticAPITooManyInitNodeDeloymentFailures
+    annotations:
+      message: Kubermatic API is failing to create too many initial node deployments.
```
Comment (xrstf) [inline config/monitoring/prometheus/rules/kubermatic-master-kubermatic.yaml]: Either the API is failing to do something, or it creates too many node deployments. But failing to create too many sounds .. like a good thing?

**Review by xrstf — CHANGES_REQUESTED** [review]
Please rephrase.

**Comment by xrstf** [comment]
Also please bump the Prometheus chart version in the Chart.yaml.

**Comment by maciaszczykm** [comment]
@xrstf Done, PTAL

**Pull request merged**